static/js/popovers.js

```diff
@@ -658,11 +651,12 @@ exports.register_click_handlers = function () {
         popovers.toggle_actions_popover(this, rows.id(row));
     });
 
-    $("#main_div").on("click", ".sender_info_hover", function (e) {
+    $("#main_div").on("click", ".sender_name,.sender_name-in-status,.inline_profile_picture", function (e) {
```

Comment:
Generally we do spaces after the commas in these multipart selectors.

Reply:
fine by me